### DISCUSSION TITLE
New install of TeamPass 3.1.1

### DISCUSSION THREAD

**Discussion by ngensea** [nilsteampassnet/TeamPass#4002] [General]
Bonjour,
Je suis en train d'installer TeamPass 3.1.1 mais aujourd'hui, je ne passe pas l'étape de vérification de la page d'installation initiale.
Un message apparait en bas à droite de la page :
![Capture d’écran du 2024-01-12 11-03-15](https://github.com/nilsteampassnet/TeamPass/assets/156420953/6124c416-6b5f-44bc-8a07-f23a69bb079e)

Et voici ce qui se passe du côté les log d'Apache :

```
[Fri Jan 12 10:05:34.455561 2024] [php:warn] [pid 1721] [client <ip client>:56146] PHP Warning:  include_once(/var/www/TeamPass/sources/../includes/config/tp.config.php): Failed to open stream: No such file or directory in /var/www/TeamPass/sources/main.functions.php on line 53, referer: http://serveur/install/install.php
[Fri Jan 12 10:05:34.455637 2024] [php:warn] [pid 1721] [client <ip client>:56146] PHP Warning:  include_once(): Failed opening '/var/www/TeamPass/sources/../includes/config/tp.config.php' for inclusion (include_path='.:/usr/share/php') in /var/www/TeamPass/sources/main.functions.php on line 53, referer: http://serveur/install/install.php
[Fri Jan 12 10:05:34.455825 2024] [php:warn] [pid 1721] [client <ip client>:56146] PHP Warning:  Undefined variable $session in /var/www/TeamPass/includes/config/settings.php on line 28, referer: http://serveur/install/install.php
[Fri Jan 12 10:05:34.455868 2024] [php:error] [pid 1721] [client <ip client>:56146] PHP Fatal error:  Uncaught Error: Call to a member function has() on null in /var/www/TeamPass/includes/config/settings.php:28\nStack trace:\n#0 /var/www/TeamPass/sources/main.functions.php(4179): require_once()\n#1 /var/www/TeamPass/sources/main.functions.php(59): loadClasses()\n#2 /var/www/TeamPass/install/install.queries.php(41): require_once('...')\n#3 {main}\n  thrown in /var/www/TeamPass/includes/config/settings.php on line 28, referer: http://serveur/install/install.php
```
Je m'interroge car je suis la procédure d'installation suivante qui fonctionne bien habituellement : https://documentation.teampass.net/#/install/installation

**Comment by jjksaji** [comment]
Hi, I  also faced the same issue. I tried in Ubuntu 22.04 and 24.04 with teamp 3.1.1 and the earlier versions also 
I also face the same issue and was not able to solve it 

On searching in the forum, I came to know that we need to cross check the 
/var/www/html/teampass/includes/config/settings.php
/var/www/html/teampass/includes/.externals/settings.php

I edited the settings.php files , gave the DB details, commented out the 
```
//if (isset($session) === true && $session->has('system-timezone') && null !== $session->get('system-timezone')) {
//    date_default_timezone_set($session->get('system-timezone'));
//}
```
Struggled with different versions of teampass and Ubuntu 22.04 and 24.04 and still I am not able solve the issues
Request guidance on what to check and how to solve this installation steps

the screen shot of the settings.php I have added here as reference 
thanks 
Joseph John 
![Screenshot 2024-05-10 at 12 09 07 PM](https://github.com/nilsteampassnet/TeamPass/assets/6416780/ea70eff7-484c-482e-a258-7731a1264c70)
